src/cli/proxy_mode.rs

```diff
@@ -1,5 +1,5 @@
-use std::{ffi::OsString, path::PathBuf};
 use std::process::ExitStatus;
+use std::{ffi::OsString, path::PathBuf};
```

Comment:
Hmmm seems like a no-op. Can we merge these imports instead?

Reply:
Done.